src/translations/sk.json

```
@@ -0,0 +1,41 @@
+{
+    "log_in": "Prihlásiť sa",
+    "log_out": "Odhlásiť sa",
+    "logged_in_as": "Prihlásený ako",
+    "logged_in": "Prihlásený",
+    "logging_in": "Prihlásenie",
+    "logging_out": "Odhlásenie",
```

Similarly like with the logging in one:

```suggestion
    "logging_out": "Prebieha odhlásenie",
```